serial.go

```diff
@@ -84,12 +86,13 @@ type Config struct {
 	// XONFlowControl bool
 
 	// CRLFTranslate bool
-	// TimeoutStuff int
+	NonBlockingRead bool
```

Comment:
Do we need this boolean?  Can we just special case the value of ReadTimeout = 0 and make that blocking?